cmd/boltzcli/commands.go

```diff
@@ -320,6 +330,29 @@ func swapInfoStream(ctx *cli.Context, id string, json bool) error {
 					fmt.Printf("Paid %dmsat routing fee, %dsat onchain fee and %dsat service fee\n", *swap.RoutingFeeMsat, *swap.OnchainFee, *swap.ServiceFee)
 					return nil
 				}
+			} else if info.ChainSwap != nil {
+				swap := info.ChainSwap
+				yellowBold.Printf("Swap Status: %s\n", swap.Status)
+
+				switch swap.State {
+				case boltzrpc.SwapState_ERROR:
+					fmt.Printf("Error: %s\n", swap.Error)
+				case boltzrpc.SwapState_REFUNDED:
+					fmt.Println("Swap was refunded")
+				}
+
+				status := boltz.ParseEvent(swap.Status)
+				switch status {
+				case boltz.SwapCreated:
+					fmt.Printf("Swap ID: %s\n", swap.Id)
+				case boltz.TransactionMempool:
+					fmt.Printf("User Transaction ID: %s\nAmount: %dsat\n", swap.FromData.GetLockupTransactionId(), swap.FromData.Amount)
+				case boltz.TransactionServerMempoool:
+					fmt.Printf("Server Transaction ID: %s\nAmount: %dsat\n", swap.ToData.GetLockupTransactionId(), swap.ToData.Amount)
```

Comment:
Maybe include the chain of the transaction in the message. Makes it easier to find. And lowercase transaction